enterprise/security/src/test/java/org/neo4j/server/security/enterprise/auth/AuthTestBase.java

```diff
@@ -235,6 +249,24 @@ void testFailListRoleUsers( S subject, String roleName, String errMsg )
                 errMsg );
     }
 
+    void testFailTestProcs( S subject )
+    {
+        assertFail( subject, "CALL test.allowedProcedure1()", READ_OPS_NOT_ALLOWED );
+        assertFail( subject, "CALL test.allowedProcedure2()", WRITE_OPS_NOT_ALLOWED );
+        assertFail( subject, "CALL test.allowedProcedure3()", SCHEMA_OPS_NOT_ALLOWED );
+//        assertFail( subject, "CALL test.allowedProcedure4()", DBMS_OPS );
```

Comment:
DBMS procedure access rights are controlled directly on the AuthSubject on a per-procedure basis, and does not map to the static modes for transactions on the database.